universite-de-bordeaux-ecole-doctorale-de-droit.csl

```diff
@@ -660,11 +660,26 @@
           <text term="ibid" text-case="capitalize-first" font-style="italic"/>
         </else-if>
         <else-if position="near-note" match="any">
-          <group delimiter=", ">
-            <text macro="author"/>
-            <text term="cited" font-style="italic"/>
-            <text macro="locator"/>
-          </group>
+          <choose>
+            <if type="legal_case">
```

Comment:
while this fixes the reported issue, what about other items commonly without authors, e.g. bills, legislation, and often websites? Could you do something like
```
<else-if variable="author editor" match="none">
  <text variable="title-short"/>
</else-if>
```
(or something along those lines that works with the logic of the style)